### PR TITLE
add utility methods

### PR DESCRIPTION
**Pull request by manuelli**
1. Adds some useful methods to ddDrake model, i.e. getting a frame using it's frameID
2. Add a few utility methods in some of the director python files that are useful for visualization and compuation of contact forces.

I have tried to add the minimal amount of changes to director that support the current work Siyuan and I are doing on the robot. Most of our director files are standalone inside openhumanoids, but these are the minimal changes to Director that we need.

### CODE REVIEW

File: src/python/director/transformUtils.py [hunk 1]
```diff
@@ -167,6 +168,8 @@ def transformFromPose(position, quaternion):
     '''
     Returns a vtkTransform
     '''
+    rotationMatrix = np.zeros((3,3))
```
Comment (mauricefallon) [inline src/python/director/transformUtils.py]: is this used?

File: src/python/director/visualization.py [hunk 2]
```diff
@@ -6,12 +6,25 @@
...
+import numpy as np
+
+
 
 import os
 import colorsys
 import weakref
 import itertools
-import numpy as np
+
```
Comment (mauricefallon) [inline src/python/director/visualization.py]: this is unused and probably better located in transformUtils

File: src/python/director/teleoppanel.py [hunk 3]
```diff
@@ -1057,7 +1057,7 @@ def __init__(self, panel, jointGroups=None):
 
         if jointGroups is None:
             # Add only these joint groups:
-            telopJointGroupNames = ['Back', 'Base', 'Left Arm', 'Right Arm', 'Neck']
+            telopJointGroupNames = ['Back', 'Base', 'Left Arm', 'Right Arm', 'Neck', "Left Leg", "Right Leg"]
```
Comment (mauricefallon) [inline src/python/director/teleoppanel.py]: this adds the left and right legs to this widget. I'd prefer not to add these but sometimes they can be useful

File: src/python/director/drcargs.py [hunk 4]
```diff
@@ -132,6 +132,29 @@ def addDefaultArgs(self, parser):
                             default=[],
                             help='Run other python startup scripts in addition to startup.py')
 
+    def getRobotType(self):
```
Comment (mauricefallon) [inline src/python/director/drcargs.py]: these are unused. 

presumably to needed to add them, but what use have you for them?

**Comment by mauricefallon** [comment]
looks quite straightforward. would you like me to merge them?

I didn't review the drake interface changes. I assume you have tested them.

**Comment by manuelli** [comment]
Thanks Maurice. This is still WIP so I can can it up a bit more before merging it. I'll try to address your comments. Some of that stuff I can get rid of.

**Comment by mauricefallon** [comment]
This code lives in a different repo (oh private)

**Pull request closed without merging**